### PR TITLE
Switch to dscale=hermite by default

### PR DESCRIPTION
**Pull request by haasn**

### CODE REVIEW

File: DOCS/man/options.rst [hunk 1]
```diff
@@ -5286,14 +5286,22 @@ them.
...
+        default for ``--dscale``.
 
     ``catmull_rom``
         Catmull-Rom. A Cubic filter in the same vein as ``mitchell``, where
         the ``B`` and ``C`` parameters are ``0.0`` and ``0.5`` respectively.
         This filter is sharper than ``mitchell``, but it results in mild
         ringing. Like ``mitchell``, this filter is good at downscaling (see
-        ``--dscale``).
+        ``--dscale``), but it is also good at upscaling (similar to and
+        slightly faster than ``lanczos``).
```
Comment (kasper93) [inline DOCS/man/options.rst:5304]: This is gross simplification. good/similar/faster are empty words. Lanczos is sharper and alias less. You made is sound like catmull_rom is faster and better.

Comment (Jules-A) [reply DOCS/man/options.rst:5304]: Yeah I wouldn't say it's similar, it has way different artifacts, maybe saying "similar sharpness" would be better. "good at upscaling" is technically true but I think Lanczos is way better for luma while catmull is better for chroma.

Comment (wiwaz) [reply DOCS/man/options.rst:5304]: I'm not sure catmull_rom can really be considered "good at upscaling" considering how aliased it is in comparison to other options, maybe the description of being "good at downscaling and upscaling" can just be removed? The filter is really just not particularly great compared to other options for both downscaling and upscaling IMO.

**Comment by Jules-A** [comment]
It's a bit inconsistent now, with hermite and catmull_rom listing their default params, mitchell is a bit left out.

**Pull request merged**